tools/report-converter/tests/unit/sparse_output_test_files/sample.out

```diff
@@ -0,0 +1 @@
+files/sample.c:3:1: warning: symbol 'machine_id' was not declared. Should it be static?
```

Comment:
Please extend this file with a new warning which contain multiline events too. Also update the `sample.c` file and the test suit.